coalib/processes/BearRunning.py

```diff
@@ -469,7 +488,7 @@ def run_global_bears(message_queue,
                                      global_result_dict))
             bearname = bear.__class__.__name__
             result = run_global_bear(message_queue, timeout, bear, dep_results,
-                                     debug=debug)
+                                     aspects, debug=debug)
```

Comment:
insert a new line with `aspects,` , so the existing line debug=debug doesnt need to be modified.
New parameters will be added before the debug parameter.